obspy/core/inventory/response.py

```diff
@@ -1324,6 +1324,31 @@ def get_sacpz(self):
         sensitivity = self.instrument_sensitivity.value
         return paz_to_sacpz_string(paz, sensitivity)
 
+    def get_paz_dict(self):
+        """
+        Get simple dictionary representation of poles, zeros and overall
+        sensitivity.
+
+        >>> from obspy import UTCDateTime, read_inventory
+        >>> t = UTCDateTime(2015, 1, 1)
+        >>> inv = read_inventory()
+        >>> resp = inv.get_response("BW.RJOB..EHZ", t)
+        >>> paz = resp.get_paz_dict()
+        >>> print(sorted(paz.keys()))
+        [u'poles', u'sensitivity', u'zeros']
```

Comment:
I'm not doing anything to these, isn't that rather a problem on a lower level.. inside the inventory types?